nameko/standalone/rpc.py

```diff
@@ -48,10 +48,17 @@ def wait(self):
         if self.exception:
             raise self.exception
 
-        if self.queue_consumer.consumer.connection is None:
+        if self.queue_consumer.stopped:
             raise RuntimeError(
                 "This consumer has been stopped, and can no longer be used"
             )
+        # TODO: seems unnecessarily unhelpful -- can't we try to reconnect?
```

Comment:
It seems not. You have to re-establish the consumer (and its exclusive reply queue) before sending any request, otherwise the reply queue may not exist when the response is published. The `MethodProxy` is used by both the standalone and dependency versions of the RPC proxy, so you can't use check the consumer connection before sending a request.